app/models/rating.rb

```diff
@@ -3,5 +3,19 @@ class Rating < Tag
   NAME = ArchiveConfig.RATING_CATEGORY_NAME
   index_name Tag.index_name
 
+  # Gives us the default ratings as Not Rated + low to high
+  def self.defaults_by_severity
+    ratings = [ArchiveConfig.RATING_DEFAULT_TAG_NAME,
+               ArchiveConfig.RATING_GENERAL_TAG_NAME, 
+               ArchiveConfig.RATING_TEEN_TAG_NAME, 
+               ArchiveConfig.RATING_MATURE_TAG_NAME, 
+               ArchiveConfig.RATING_EXPLICIT_TAG_NAME]   
+    ratings_by_id = Rating.where(name: ratings).inject({}) do |result, rating| 
+      result[rating.name] = rating
+      result
+    end
+    sorted_ratings = ratings.map {|id| ratings_by_id[id]}
```

Comment:
Useless assignment to variable - `sorted_ratings`.<br>Space between { and | missing.<br>Space missing inside }.